### PR TITLE
Sending test cases results

### PR DESCRIPTION
**Pull request by Fgerthoffert**

### CODE REVIEW

File: src/commands/zencrepes.ts [hunk 1]
```diff
@@ -103,6 +111,31 @@ class JahiaTestrailReporter extends Command {
...
+            caseSuccess: test.status === 'PASS' ? 1 : 0,
+            caseFailure: test.status === 'FAIL' ? 1 : 0,
+            createdAt: r.timestamp === undefined ? new Date().toISOString() : new Date(r.timestamp).toISOString(),
+          })
+        }
+      }
+    }
+    // const testCases = report.reports.reduce((acc: any, report: any) => {
+    //   console.log(report)
+    // }, [])
```
Comment (rknj) [inline src/commands/zencrepes.ts:137]:
```suggestion
```

File: src/global.type.ts [hunk 2]
```diff
@@ -42,6 +43,15 @@ export interface JRRun {
   reports: JRReport[];
 }
 
+// A run is composed of multiple junit files
+export interface JRCase {
+  id: string;
+  name: string;
+  suite: string;
+  duration: number;
+  statis: string;
```
Comment (rknj) [inline src/global.type.ts:52]:
```suggestion
  state: string;
```

Comment (rknj) [reply src/global.type.ts:52]: according to its usage in `const zcPayload: ZenCrepesStateNode`
Also, aren't we missing these: jahia, module, caseTotal, caseSuccess, caseFailure and createdAt?

**Review by rknj — CHANGES_REQUESTED** [review]

File: src/global.type.ts (hunk 2 above)
Comment (Fgerthoffert) [reply src/global.type.ts]: thanks, I added the missing ones and corrected the typo

**Review by rknj — APPROVED** [review]

**Pull request merged**